src/console/plugin.ts

```diff
@@ -319,7 +319,7 @@ namespace Private {
       `Name: ${label}\n` +
       `Directory: ${ContentsManager.dirname(path)}\n` +
       `Kernel: ${displayName}\n` +
-      `Opened: ${dateTime(opened.toISOString())}`
+      `Connected: ${dateTime(opened.toISOString())}`
```

Comment:
How about changed the variable name from `opened` as well, since it's got different semantics?